region/crateria/east/East Ocean.json

```diff
@@ -898,7 +919,12 @@
           "canWalljump",
           "HiJump",
           "SpaceJump",
-          "canIBJ"
+          "canIBJ",
+          "canSpringBallJumpMidAir",
+          {"and": [
+            "canTrickyJump",
```

Comment:
pretty easy to do from the ledge, maybe not obvious?

Reply:
it's not too hard, but I felt like it was harder than `Medium`. Maybe it's ok with a careful jump, since it will only be enabled with collectible wall jump?